### DISCUSSION TITLE
Translation error on the start page

### DISCUSSION THREAD

**Discussion by ASSANDHOLE** [community/community#40718] [Mobile]
The 一键离开 here means one key to get away instead of I'm guessing one key away, in Chinese 一键之遥. 
![image](https://user-images.githubusercontent.com/63102113/205460964-a5e2c957-e757-47a9-90e8-701d0028c3a7.jpeg)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.91.0 (85250348)
Device: iPhone 12 Pro Max (iOS 16.1.2)
</details>

**Comment by ossanna16** [comment]
Hi there @ASSANDHOLE and welcome to our community! Thank you for reporting this!

**Comment by aconsuegra** [comment]
Thanks for your report @ASSANDHOLE. We have just fixed the issue, and it will be included in one of our upcoming releases.
